db/migrate/20161116114622_remove_invalid_draft_content_item.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "helpers/delete_content_item"
+
+class RemoveInvalidDraftContentItem < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
Am I right to think that this migration would not be reversible after you destroy this content_item?

http://api.rubyonrails.org/classes/ActiveRecord/IrreversibleMigration.html